src/store/configureStore.js

```diff
@@ -1,17 +1,25 @@
-import { applyMiddleware, compose, createStore } from 'redux';
-import createSagaMiddleware from '@redux-saga/core';
-import rootReducer from './reducers/rootReducer';
-import { rootSaga } from './sagas';
+import {createBrowserHistory} from "history";
+import {applyMiddleware, combineReducers, compose, createStore} from "redux";
+import {connectRouter, routerMiddleware} from "connected-react-router";
+import thunkMiddleware from "redux-thunk";
+import mainReducer from './reducers/mainReducer'
+
+export const history = createBrowserHistory()
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-const sagaMiddleware = createSagaMiddleware();
+const rootReducers = combineReducers({
```

Comment:
rootReducers  можно  понять как массив reducer' ов
```suggestion
const rootReducer = combineReducers({
```